docs/Deploying/Server-Installation.md

```diff
@@ -420,11 +420,11 @@ If you go back to [https://localhost:8080/hosts/manage](https://localhost:8080/h
 
 ## Deploying Fleet on Kubernetes
 
-In this guide, we're going to focus on deploying Fleet only on a Kubernetes cluster. Kubernetes is a container orchestration tool that was open sourced by Google in 2014.
+In this guide, we will focus on deploying Fleet only on a Kubernetes cluster. Kubernetes is a container orchestration tool that was open sourced by Google in 2014.
 
 We will assume you have `kubectl` and MySQL and Redis are all set up and running. Optionally you have minikube to test your deployment locally on your machine.
 
-To deploy the Fleet server and connect to its dependencies(MySQL and Redis) we will set up a `deployment.yml` file with the following specifications:
+To deploy the Fleet server and connect to its dependencies(MySQL and Redis), we will set up a `deployment.yml` file with the following specifications:
 
 ```yaml
```

Comment:
@Desmi-Dizney are you referring to `yaml` on line 429? If so, I think we do want to keep this lowercase.

We want to keep `yaml` lowercase because this is a keyword that GitHub uses to render the correct syntax highlighting.

@DominusKelvin do you know if there's a reason to keep `fleet` lowercase in the example `deployment.yml` file? Specifically on line 440, 444, and 447.

Reply:
@DominusKelvin, whenever you get a moment, let me know on this one.

Reply:
@Desmi-Dizney @noahtalerman Just my two cents, but I think that since most of us are used to code being camelcase, it makes sense to stick with that in the code examples. If for no other reason than that `fleet` feels a lot more natural than `Fleet` in this context to type out and since it's happening multiple times there's a good chance an `f` would slip in there somewhere.

Reply:
And I'm sure I'm not the only one that avoids copy/paste from pretty much anywhere for blocks of code because VSC and invisible characters have burned me three too many times.

Reply:
Makes sense! Thanks Kathy.